docs/UserGuide.adoc

```diff
@@ -540,54 +540,51 @@ Shows the 2nd version of offline copy of bookmark 1. If it is annotated,
 then annotations are shown as well.
 
 
-=== Adding reminder: `reminder`
+=== Adding a reminder: `reminder`
 
-Adds a reminder to visit a bookmark. You can set the time for the reminder. The app
-will inform you at that time. You can also choose whether to open the bookmark.
-The default setting is open.
+Adds a reminder to visit a bookmark. You can set the time for the reminder.
 
 Format: `reminder INDEX t/TIME_DATE [n/NOTE=Open]`
 
 * INDEX is the index of the bookmark that you want to add with a reminder.
  * TIME_DATE follows HHMM dd/mm/yyyy format.
-* NOTE is the setting of whether you want to open the bookmark.
+* NOTE is the description of the reminder, the default value is Open.
 
 Examples:
 
 * `reminder 1 t/2359 02/07/2019` +
-Adds a reminder for bookmark 1, and set the time at 02/07/2019. The
-reminder will remind you at 23:59 on 2nd July, 2019.
-It will also open the bookmark for you.
+Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
+It reminds you to open the bookmark.
 
-* `reminder 1 t/2359 02/07/2019 n/Close` +
-Adds a reminder for bookmark 1, and set the time at 23:59 of 02/07/2019. The reminder will
-remind you at the time and would not open the bookmark for you.
+* `reminder 1 t/2359 02/07/2019 n/Read announcements` +
+Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
+The reminder reminds you to read announcements.
 
-=== Editing reminder: `reminder edit`
-Edit a reminder at the reminder's index. You can edit either time or whether to open it.
+
+
+=== Editing a reminder: `reminder edit`
+Edits the reminder of the specified bookmark. You can edit either the time or the note of the reminder.
 
 Format: `reminder edit INDEX [t/TIME_DATE] [n/NOTE]`
-* INDEX is the index of the reminder that you want to edit.
+* INDEX is the index of the bookmark that has the reminder that you want to edit.
```

Comment:
Following the discussion on Telegram: `INDEX` should be the index of the reminder, correct?

Reply:
Okay